exporter/azure_monitor/azure_monitor.go

```diff
@@ -0,0 +1,46 @@
+package azure_monitor
+
+import (
+	"fmt"
+	"errors"
+	"go.opencensus.io/exporter/azure_monitor/common"
+	"go.opencensus.io/trace"
+)
+
+type AzureTraceExporter struct {
+	projectID          string
+	InstrumentationKey string
+	options            common.Options
+}
+
+func NewAzureTraceExporter(o common.Options) (*AzureTraceExporter, error) {
+	if o.InstrumentationKey == "" {
+		return nil, errors.New("missing Instrumentation Key for Azure Exporter")
+	}
+	e := &AzureTraceExporter {

```

Comment:
Use a meaningful name rather than `e`.

Reply:
Nice catch. Yes, I will replace `e` with `exporter` and replace `o` with `options`